CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/SplitNodesPlugin.java

```diff
@@ -152,6 +152,25 @@ private void editResultStore(final RecordStore result, final String left, final
             result.set(GraphRecordStoreUtilities.SOURCE + AnalyticConcept.VertexAttribute.TYPE, leftVertexTypesMatches.get(0));
         }
         result.set(GraphRecordStoreUtilities.DESTINATION + VisualConcept.VertexAttribute.IDENTIFIER, right);
+        
+        //Loops through all of the Node attributes and copies them to the new node
+        query.reset();
+        while(query.next()){
+            for (final String key : query.keys()){
+                if(SOURCE_ID.replace("<string>", "").equals(key) || SOURCE_IDENTIFIER.replace("<string>", "").equals(key)) {
+                     //Skips the id and Identifier to make the new node unique
+                } else if ((GraphRecordStoreUtilities.SOURCE + VisualConcept.VertexAttribute.X).equals(key) 
+                        || (GraphRecordStoreUtilities.SOURCE + VisualConcept.VertexAttribute.Y).equals(key) 
+                        || (GraphRecordStoreUtilities.SOURCE + VisualConcept.VertexAttribute.Z).equals(key)){ 
+                    //The coordinates are also skipped so that the second node is not created in the exact same location
+                    //as the first node
+                } else {
+                     result.set(GraphRecordStoreUtilities.DESTINATION + key.replace(GraphRecordStoreUtilities.SOURCE, ""), 
```

Comment:
A very minor comment but I would put this in 1 line because it is a little confusing seeing the `query.get` on its own.